README.md

```diff
@@ -24,6 +24,26 @@ tls_interop --client ${NSS_ROOT}/dist/Darwin15.6.0_cc_64_DBG.OBJ/bin/nss_bogo_sh
 To swap client and server, you need to run it twice.
 
 
+Cargo Test Instructions
+============================
+Some of the internal rust test cases run with "cargo test" assume readily built
+versions of nss, boringssl and openssl being available in the parent diretory.  
+The NSS shim is expected to be found at "../dist/Debug/bin/nss_bogo_shim".  
+The BoringSSL shim is expected to be found at "../boringssl/build/ssl/test/bssl_shim".  
+The OpenSSL shim is expected to be found at "../openssl/tests/ossl_shim/ossl_shim".  
 
+All three default paths can be overwritten by setting the following environment variables:  
+NSS_SHIM_PATH = ${NSS_ROOT}/bin/nss_bogo_shim  
+BORING_ROOT_DIR = ${BORINGSSL_ROOT}  
+OSSL_SHIM_PATH = ${OPENSSL_ROOT}/tests/ossl_shim/ossl_shim  
 
+```
+cargo test
+```
+Runs only a set of very basic connection tests between nss and the other two shims. 
 
+```
+cargo test -- --ignored
```

Comment:
Why are these other tests ignored?

Reply:
My idea was to have separate ways to run either the basic connection tests (to see if interop works in general) or the full set of tests from the cases.json file, that is supposed to grow signitficantly.
Do you think this is necessary at all, and if yes, should I find a nicer way to do that than to use the --ignored flag?

Reply:
Hm, I think I'd run all tests for now. If you want to run only a subset of tests you can do like this https://doc.rust-lang.org/book/second-edition/ch11-02-running-tests.html#filtering-to-run-multiple-tests.
The `--ignored` flag should probably only be used for tests that are known to be slow or broken.